### ISSUE TITLE
Some form of onboarding experience

### ISSUE THREAD

**Issue by waydabber** [MonitorControl/MonitorControl#676]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

First time users may not clearly understand what this app does, how does it work, etc. The first thing they encounter is a popup message and some complaints about missing permissions in Privacy. This could be improved with a simple onboarding experience on first run.

### Describe the solution you'd like

Just a nice welcome window with 2-3 sliding pages explaining some basic stuff, next steps, etc.

### Describe alternatives you've considered

Not having onboarding. :)

**Comment by the0neyouseek** [comment]
I completely agree, and I also agree that it should not be more than 3 slides (and provide a `Skip` action)

**Comment by bandtank** [comment]
Even a few comments in the readme would suffice. On that note, the readme doesn't even say how to run the program. I built it in xcode, but now what? A few sentences about how to go from start to finish in the readme would be extremely little effort. It is pretty frustrating that nothing is written about OP's request or anything else. "Clone and open in xcode" as the only instructions is a little hard to believe - especially considering how much time was spent developing this project.

**Comment by waydabber** [comment]
@bandtank - yes, the readme should be improved. But the main method of using the app is to download a [release](https://github.com/MonitorControl/MonitorControl/releases/download/v4.0.0-beta2/MonitorControl.4.0.0-beta2.dmg) and install it as any normal Mac app.

Building it in XCode is mainly for developers and they will probably have no problem doing it. I can't think of much else in terms of instruction for a developer than clone the project, open it and then press build and solve the issues that might arise meanwhile as XCode will clearly complain if it has a problem.

**Comment by waydabber** [comment]
Probably we should place the Download link in a more obvious location, now that I look at it with fresh eyes, it's not exactly emphasized. :)

**Comment by waydabber** [comment]
@bandtank - the README was updated a bit with additional information on the app and a prominent Download app badge.

**Comment by the0neyouseek** [comment]
Here's an onboarding screen I designed, feel free to tell what you think
![Onboarding](https://user-images.githubusercontent.com/4009175/149617598-90fdca18-e3cc-4747-9565-91f87cb81de3.jpg)

**Comment by waydabber** [comment]
I love it! Looks super cool!

**Comment by waydabber** [comment]
I like the new onboarding screen very much! Well done!

Maybe just one suggestion/idea (for the future): we might want to let the user know that after `Start using MonitorControl` the app can be reached via the menu extra. There were some feedbacks previously if I remember correctly that this was not 100% obvious for some.

**Comment by the0neyouseek** [comment]
Maybe simply opening the menu would be enough. What do you think ?

**Comment by waydabber** [comment]
Yes, that makes things obvious. Neat solution.

**Comment by waydabber** [comment]
Hey @the0neyouseek 

I added a Hungarian translation PR for the onboarding.

Additionally I found two minor issues that might need a bit of an investigation:

- when clicking on `Start using MonitorControl` the menu automatically opens - but when this happens, the menu sliders, keyboard etc do not actually work - which is not an ideal first experience imho. :)
- When the app already has the required privileges, the `Open System Preferences` button does nothing. I think in this case the button should not present itself in the first place or we should tell the user that the app already has the privileges so no need to do anything. This is a rare issue, might happen only on a manual preferences reset I think or when the user (based on some tutorial) adds MonitorControl to the list even before first firing up the app - so this is really a low priority issue, just saying it for completeness' sake.
